velox/common/memory/Memory.h

```diff
@@ -111,6 +111,11 @@ struct MemoryManagerOptions {
   /// during the memory arbitration.
   uint64_t memoryPoolTransferCapacity{32 << 20};
 
+  /// Specifies the max time to wait for memory reclaim by arbitration. The
+  /// memory reclaim might fail if the max wait time has exceeded. If it is
+  /// zero, then there is no timeout.
+  uint64_t memoryReclaimWaitMs{0};
```

Comment:
Would it be safer to have a non-zero default? e.g. 5 min?

Reply:
Yeah, we could.